api.go

```diff
@@ -492,8 +492,8 @@ func (d *Context) GetLength(index int) int {
 }
 
 // See: http://duktape.org/api.html#duk_get_lstring
-func (d *Context) GetLstring(index int, outLen int) string {
-	if s := C.duk_get_lstring(d.duk_context, C.duk_idx_t(index), (*C.duk_size_t)(unsafe.Pointer(&outLen))); s != nil {
+func (d *Context) GetLstring(index int, outLen *int) string {
```

Comment:
```
func (d *Context) GetLstring(index int) (string, int) {
  var outLen int
  if s := C.duk_get_lstring(d.duk_context, C.duk_idx_t(index), (*C.duk_size_t)(unsafe.Pointer(&outLen))); s != nil {
    return C.GoString(s), outLen
  }
  
  return "", 0
}
```
Isn't it would be better, you don't need to provide outLen because it would be obtain as a return value.